FlexMeasures/flexmeasures
# Audit log

## Description

Added AuditLog data model
Started adding new records to it on user creation / deletion / activating etc.
New "Audit Log" buttons on account and user page that lead to audit log page

## Look & Feel

New endpoints:
 - GET /accounts/auditlog/<account_id>
 - GET /users/auditlog/<user_id>

## How to test

Run `flexmeasures db upgrade`
Create new user / delete existing one / activate or deactivate some user
Check there are records on audit log page for this user

## Further Improvements

Fix bug with user not being able to view himself unless he is an admin.

## Related Items

-

---
